server/lib/page-type-util.js

```diff
@@ -0,0 +1,126 @@
+const PageTypes = {
+    PAGE: 'PAGE',
+    PRODUCT: 'PRODUCT',
+    CATEGORY: 'CATEGORY',
+    BRAND: 'BRAND',
+    ACCOUNT_RETURN_SAVED: 'ACCOUNT_RETURN_SAVED',
+    ACCOUNT_ADD_RETURN: 'ACCOUNT_ADD_RETURN',
+    ACCOUNT_RETURNS: 'ACCOUNT_RETURNS',
+    ACCOUNT_ADD_ADDRESS: 'ACCOUNT_ADD_ADDRESS',
+    ACCOUNT_ADD_WISHLIST: 'ACCOUNT_ADD_WISHLIST',
+    ACCOUNT_WISHLISTS: 'ACCOUNT_WISHLISTS',
+    ACCOUNT_WISHLIST_DETAILS: 'ACCOUNT_WISHLIST_DETAILS',
+    ACCOUNT_EDIT: 'ACCOUNT_EDIT',
+    ACCOUNT_ADDRESS: 'ACCOUNT_ADDRESS',
+    ACCOUNT_INBOX: 'ACCOUNT_INBOX',
+    ACCOUNT_DOWNLOAD_ITEM: 'ACCOUNT_DOWNLOAD_ITEM',
+    ACCOUNT_ORDERS_ALL: 'ACCOUNT_ORDERS_ALL',
+    ACCOUNT_ORDERS_INVOICE: 'ACCOUNT_ORDERS_INVOICE',
+    ACCOUNT_ORDERS_DETAILS: 'ACCOUNT_ORDERS_DETAILS',
+    ACCOUNT_ORDERS_COMPLETED: 'ACCOUNT_ORDERS_COMPLETED',
+    ACCOUNT_RECENT_ITEMS: 'ACCOUNT_RECENT_ITEMS',
+    AUTH_ACCOUNT_CREATED: 'AUTH_ACCOUNT_CREATED',
+    AUTH_LOGIN: 'AUTH_LOGIN',
+    AUTH_CREATE_ACC: 'AUTH_CREATE_ACC',
+    AUTH_FORGOT_PASS: 'AUTH_FORGOT_PASS',
+    AUTH_NEW_PASS: 'AUTH_NEW_PASS',
+    BLOG_POST: 'BLOG_POST',
+    BLOG: 'BLOG',
+    BRANDS: 'BRANDS',
+    CART: 'CART',
+    COMPARE: 'COMPARE',
+    CONTACT_US: 'CONTACT_US',
+    HOME: 'HOME',
+    GIFT_CERT_PURCHASE: 'GIFT_CERT_PURCHASE',
+    GIFT_CERT_REDEEM: 'GIFT_CERT_REDEEM',
+    GIFT_CERT_BALANCE: 'GIFT_CERT_BALANCE',
+    ORDER_INFO: 'ORDER_INFO',
+    SEARCH: 'SEARCH',
+    SITEMAP: 'SITEMAP',
+    SUBSCRIBED: 'SUBSCRIBED',
+    UNSUBSCRIBE: 'UNSUBSCRIBE',
+};
+
+const templateFileToPageTypeMap = {
```

Comment:
Really wish we could fetch this from somewhere or include it as a library or something.

Reply:
This mapping is in Storefront also now, so we technically are doing this in 3 places. BCApp, Storefront and now in Stencil CLI 🤷‍♂️

Reply:
Are we in hurry with this ticket, so we are introducing here tech debt? If so, lets create a ticket to address it by adding an endpoint to Storefront (or other more relevant service)